cobra_commander-yarn/README.md

```diff
@@ -0,0 +1,33 @@
+# CobraCommander::Yarn
+
+Welcome to your new gem! In this directory, you'll find the files you need to be able to package up your Ruby library into a gem. Put your Ruby code in the file `lib/cobra_commander/yarn`. To experiment with that code, run `bin/console` for an interactive prompt.
+
+TODO: Delete this and the text above, and describe your gem
+
+## Installation
+
+Install the gem and add to the application's Gemfile by executing:
+
+    $ bundle add cobra_commander-yarn
+
+If bundler is not being used to manage dependencies, install the gem by executing:
+
+    $ gem install cobra_commander-yarn
+
+## Usage
+
+TODO: Write usage instructions here
```

Comment:
Maybe address the TODOs in the two READMEs? Anything brief will do.

Reply:
Forgot about the new README's, thanks.